frontend/views/pages/Contributions.vue

```diff
@@ -299,6 +260,9 @@ export default {
     },
     displayName (username) {
       return this.globalProfile(username).displayName || username
+    },
+    wCurrency (amount) {
```

Comment:
I like how the other file called this function `withCurrency`, can we make this also clearer by naming it `withCurrency`?